pkg/service/backup/worker_clustercheck.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2022 ScyllaDB
+
+package backup
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/scylladb/scylla-manager/v3/pkg/util/retry"
+)
+
+// CheckCluster checks the current load on every element of hosts and returns the first one exceeding the configured
+// maximum threshold. See [Config.MaxLoadThreshold].
+//
+// If [Config.MaxLoadThreshold] was set to 100 the method is no-op.
+func (w *worker) CheckCluster(ctx context.Context, hosts []hostInfo, clusterCheckBackoff retry.Backoff) error {
+	if w.Config.MaxLoadThreshold == 100 {
+		return nil
+	}
+	notify := func(err error, wait time.Duration) {
+		w.Logger.Info(ctx, "Current load of one of the nodes is above the configured threshold...", "error", err, "wait", wait)
+	}
+
+	return retry.WithNotify(ctx, func() error {
+		for _, h := range hosts {
+			load, err := w.Client.CurrentLoad(ctx, h.IP)
+			if err != nil {
+				return retry.Permanent(errors.Wrap(err, "node's load metric"))
```

Comment:
So if we can't extract metrics of only one node, the validation of the others should be skipped? Do we want such behavior? 

Moreover, this approach can return different results for different orders of nodes in the slice.
Let's assume that the slice contains both node that exceeds load param and node from which we cannot extract load metrics. Then, depending on which node is examined first, we return different errors.
This in combination with the fact, that decision about proceeding with the backup depends on the type of returned error, can create some issues.

I would suggest that if we can't extract metrics of a given node, we just skip it's validation, but we proceed with the validation of the remaining nodes and if any node exceeds load limit, we do not proceed with the backup.